pnpm-lock.yaml

```diff
@@ -12909,7 +12942,6 @@ packages:
   /source-map@0.7.4:
     resolution: {integrity: sha512-l3BikUxvPOcn5E74dZiq5BGsTb5yEwhaTSzccU6t4sDOH8NWJCstKO5QT2CvtFoK6F0saL7p9xHAqHOlCPJygA==}
     engines: {node: '>= 8'}
-    dev: true
```

Comment:
These dependencies lost their `dev` status because `packages/angular_devkit/core` requires them as non-dev. Perfectly fine!

**but** this brings up a good future opportunity in ensuring the root `package.json` matches the dependencies in the "shipped package.json". cc. @clydin. I assume this _would_ work if we could use pnpm for publishing as it would automatically e.g. replace `workspace:XXX` with the proper versions.